File: TextosJapones/src/main/java/org/jisho/textosJapones/database/dao/KanjiDao.java
package org.jisho.textosJapones.database.dao;

import org.jisho.textosJapones.model.entities.Kanji;
import org.jisho.textosJapones.model.exceptions.ExcessaoBd;

public interface KanjiDao {
    Kanji select(String kanji) throws ExcessaoBd;

}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/database/dao/KanjiDao.kt
package br.com.fenix.processatexto.database.dao

import br.com.fenix.processatexto.model.entities.processatexto.Kanji
import java.sql.SQLException
import java.util.*


interface KanjiDao {
    @Throws(SQLException::class)
    fun select(kanji: String): Optional<Kanji>
}